pkg/skaffold/schema/latest/config.go

```diff
@@ -670,6 +670,19 @@ type DeployType struct {
 
 	// KustomizeDeploy *beta* uses the `kustomize` CLI to "patch" a deployment for a target environment.
 	KustomizeDeploy *KustomizeDeploy `yaml:"kustomize,omitempty"`
+
+	// CloudRunDeploy *alpha* deploys to Google Cloud Run using the Cloud Run v1 API
+	CloudRunDeploy *CloudRunDeploy `yaml:"cloudrun,omitempty"`
+}
+
+// CloudRunDeploy *alpha* deploys the container to Google Cloud Run.
+type CloudRunDeploy struct {
+	// ProjectID of the GCP Project to use for Cloud Run.
+	DefaultProjectID string `yaml:"defaultprojectid,omitempty"`
```

Comment:
Another schema will have to be cut to add these variables properly, see:
https://github.com/GoogleContainerTools/skaffold/blob/main/DEVELOPMENT.md#making-a-config-change